spec/unit/provider/package/dpkg_spec.rb

```diff
@@ -52,8 +51,8 @@
   end
 
   before(:each) do
-    allow(provider).to receive(:shell_out!).with("dpkg-deb -W #{new_resource.source}", timeout: 900).and_return(dpkg_deb_status)
-    allow(provider).to receive(:shell_out).with("dpkg -s #{package}", timeout: 900).and_return(dpkg_s_status)
+    allow(provider).to receive(:shell_out!).with("dpkg-deb -W #{source}", timeout: 900).and_return(dpkg_deb_status)
```

Comment:
That actually blows up on me in a way that makes it look like the allow()'s aren't working...

And if I just add this line at the end of the before:

```
provider.shell_out!("invalid stuff")
```

it already blows up with:

```
        #<Chef::Provider::Package::Dpkg:0x007faf4ac51038> received :shell_out! with unexpected arguments
          expected: ("dpkg-deb -W /tmp/wget_1.11.4-1ubuntu1_amd64.deb", {:timeout=>900})
               got: ("invalid stuff")
```